controllers/configurationpolicy_controller.go

```
@@ -3698,6 +3706,19 @@ func (r *ConfigurationPolicyReconciler) alreadyEvaluated(
 	return alreadyEvaluated, resultTyped.compliant, resultTyped.msg
 }
 
+// getEvalObjKey returns a key for the cached policy map based on the
+// object UID and hash of the object template. This allows multiple object templates
+// to evaluate the same object without overwriting the cache.
+func getEvalObjKey(objUID types.UID, objectT *policyv1.ObjectTemplate) string {
+	if objectT == nil {
+		return fmt.Sprintf("%s", objUID)
```

golangci-lint (2.11.4)

[error] S1025: the argument's underlying type is a string, should use a simple conversion instead of fmt.Sprintf

(staticcheck)